exampleSite/content/awesome-team/gator.md

```diff
@@ -10,7 +11,6 @@ selfLink = "member-fragment"
 title = ""
 subtitle = ""
```

Comment:
This data shouldn't be in the specific member files. You already removed it from the others, so probably just slipped through.